components/search-box.js

```diff
@@ -0,0 +1,35 @@
+import { Input } from '@carbonplan/components'
+import { Box, Flex, Text } from 'theme-ui'
+
+export const SearchBox = () => {
+  return (
+    <>
+      <Box as='section'>
+        <Flex
+          sx={{
+            alignItems: 'flex-end',
+          }}
+        >
+          <Text
+            sx={{
+              fontWeight: 'bold',
+              fontSize: 3,
+            }}
+          >
+            Search
+          </Text>
+          <Box
+            sx={{
+              flexGrow: 1,
+              ml: 3,
+            }}
+          >
+            <form>
+              <Input placeholder='Enter your search term...' size='xs' />
```

Comment:
The placeholder text is consistently getting cut off. We should probably make the input wider (it's 3 columns wide in the Figma mocks), but we could also shorten or remove the placeholder.